app/controllers/wopi_controller.rb

```diff
@@ -2,7 +2,7 @@ class WopiController < ActionController::Base
   include WopiUtil
 
   before_action :load_vars, :authenticate_user_from_token!
-  before_action :verify_proof!
+  before_action :verify_proof! unless Rails.env.test?
```

Comment:
We could use either `skip_before_action` call in tests or
```
allow(controller).to receive(:verify_proof!).and_return(true)
```

both of which are less invasive.

Reply:
Thank you, it is working.